### DISCUSSION TITLE
Registry - Configure caching

### DISCUSSION THREAD

**Discussion by maheshkumargangula** [Sunbird-RC/community#127] [Issues]
@dileepbapat - We have redis in our deployment scripts. But, I don't find the configuration details with sunbird registry microservice. Subird Registry supports caching? If yes, please share the details about how to enable caching and configure or tune based on a specific registry instance requirement. Like, changing the TTL, etc.

Cc: @vrayulu

**Comment by pramodkvarma** [comment]
Adding @tejash-jl in case @dileepbapat is tied up. I had spoken to Dileep about this during DIVOC build. Not sure if ever implemented caching for recently/frequently used certs. I am curious too.

**Comment by dileepbapat** [comment]
Yes redis is used for caching certificates, we are restructuring the modules to pull relevant certification modules in to services folder, will update once that is done.

here is the reference from [divoc](https://github.com/egovernments/DIVOC/blob/main/backend/certificate_signer/config/config.js#L24)
